config/sampleCfg_TauTau_UL18.cfg

```diff
@@ -0,0 +1,185 @@
+[samples]
```

Comment:
could you also just rename this file to make it clear it contains the `eos` file paths please?

Reply:
configs were renamed in the latest commit.